bin/opencanaryd

```diff
@@ -18,6 +18,16 @@ function usage() {
 
 }
 
+# Use sudo when not running as root
+function sudo() {
+  if [ "$EUID" -ne 1 ]; then
```

Comment:
```
$ sudo su
$ echo $EUID
0
```
This is on ubuntu 22.04. Im nervous of the `$EUID` check because im unsure how ubiquitous it is.

Reply:
Ah my bad, It should have been `0`, I used `1` for testing and missed to fix it before committing 🤦‍♂️